src/ORM/Query/SelectQuery.php

```diff
@@ -1795,3 +1795,7 @@ public function isAutoFieldsEnabled(): ?bool
         return $this->_autoFields;
     }
 }
+
+// phpcs:disable
+class_exists('Cake\ORM\Query');
```

Comment:
I thought we would have dropped ORM\Query in 5.0, it's been deprecated since 4.4.

Reply:
Err my bad, it's deprecated since 4.5 not 4.4.